plugins/module_utils/network/sonic/config/bgp_neighbors_af/bgp_neighbors_af.py

```diff
@@ -646,6 +809,19 @@ def get_delete_bgp_neighbors_af_requests(self, commands, have, is_delete_all):
             requests.extend(self.get_delete_single_bgp_neighbors_af_request(cmd, is_delete_all, match))
         return requests
 
+    def sort_lists_in_config(self, config):
+        if config:
+            config.sort(key=lambda x: x['vrf_name'])
```

Comment:
How about key "bgp_as", if not used, it is not proper sort.

Reply:
Included "bgp_as" as part of sorting.